projects/ion/src/lib/skeleton/skeleton.component.html

```diff
@@ -0,0 +1,8 @@
+<span
+  [style.width.px]="width"
+  [style.height.px]="height"
+  [style.borderRadius]="getRadius()"
```

Comment:
call a function here can come lose performance, my suggestion is add your logic here